Sources/Rosetta/PlayMode/CardSets/BlackTempleCardsGen.cpp

```diff
@@ -706,6 +706,9 @@ void BlackTempleCardsGen::AddPaladin(std::map<std::string, CardDef>& cards)
     // --------------------------------------------------------
     // Text: Give a minion +2/+2. Draw a card.
     // --------------------------------------------------------
```

Comment:
You should add comments for the information of play requirements.
```suggestion
    // --------------------------------------------------------
    // PlayReq:
    // - REQ_TARGET_TO_PLAY = 0
    // - REQ_MINION_TARGET = 0
    // --------------------------------------------------------    
```